### PR TITLE
APPS-2448 Hotfix - Release Nextaur 1.7.3

### PR DESCRIPTION
**Pull request by r-i-v-a**
Jira = APPS-2448

### CODE REVIEW

**Review by YuxinShi0423 — CHANGES_REQUESTED** [review]
Thanks Riva for the fix! Shall we update the changelog as well?

**Comment by r-i-v-a** [comment]
APPS-2458 not mentioned in changelog since it was introduced and fixed in the same (dxpy) version. @YuxinShi0423

**Comment by r-i-v-a** [comment]
You may need to edit the beta version number, as well @kpjensen

**Review by kpjensen — CHANGES_REQUESTED** [review]
Please add the updated changelog items under the `Unreleased` section. These changes will not be part of 0.370.1. They will be included in a new beta release 0.370.2.

**Review by kpjensen — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ Categories for each release: Added, Changed, Deprecated, Removed, Fixed, Securit
 
 ## Unreleased
 
+## [370.2] - beta
```
Comment (mhrvol) [inline CHANGELOG.md:9]: I believe we usually do not specify the version here - we usually just put everything into "unreleased" and Kurt changes it to a version during the deploy. But as he approved I guess this is not an issue.

**Comment by r-i-v-a** [comment]
GH still says changes requested -- @YuxinShi0423 can you please review

**Review by YuxinShi0423 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (r-i-v-a) [reply CHANGELOG.md:9]: Good to know, next time I'll just leave ## Unreleased

**Pull request merged**